commands/new_function_test.go

```diff
@@ -5,37 +5,334 @@ package commands
 
 import (
 	"os"
+	"reflect"
 	"regexp"
 	"testing"
 
+	"github.com/openfaas/faas-cli/stack"
 	"github.com/openfaas/faas-cli/test"
 )
 
-func Test_new(t *testing.T) {
-	//TODO activate the teardown when PR#87 is merged defer teardown()
-	funcName := "test-function"
+const SuccessMsg = `(?m:Function created in folder)`
```

Comment:
When you capitalize variables in Go it has unexpected behavior of making them available globally and external to the package. Best to put these in the tests and repeat them for readability (we don't want to be jumping around) and when we absolutely have to use consts etc they should start with lowercase.